src/settings.cpp

```diff
@@ -189,7 +215,18 @@ settings::settings(config::settings context)
                 0xac, 0x00, 0x00, 0x00, 0x00});
             bip65_freeze = 1351;
             bip66_freeze = 1251;
+            bip34_freeze = 0;
             bip16_activation_time = 0x4f3af580;
+            const auto genesis_checkpoint = config::checkpoint(
+                "06226e46111a0b59caaf126043eb5bbf28c34f3a5e332a1fc7b2b73cf188910f",
+                0);
```

Comment:
use `genesis_block.hash()` ?

and avoid the copy:

```cpp
            const config::checkpoint genesis_checkpoint(genesis_block.hash(), 0);
```

Reply:
Good point, done!